src/keys/get-management-token.ts

```diff
@@ -142,8 +139,12 @@ export const createGetManagementToken = (
  * @category Keys
  */
 export const getManagementToken = (privateKey: string, opts: GetManagementTokenOptions) => {
-  return createGetManagementToken(createLogger({ filename: __filename }), createHttpClient())(
-    privateKey,
-    opts
-  )
+  if ((opts.reuseToken || opts.reuseToken === undefined) && !defaultCache) {
+    defaultCache = new NodeCache()
+  }
+  return createGetManagementToken(
```

Comment:
I guess we should now ask why this function even exists. We could just inline the `createGetManagementToken` into this function, and it probably makes the code slightly easier to follow. I don't think we need to do it in this PR though.